src/js/crypto.js

```diff
@@ -61,3 +61,98 @@ exports.randomBytes = function(size, callback) {
   }
   return buf;
 };
+
+
+// Sign & Verify
+var _Sign = require('crypto_sign').Sign;
+var _Verify = require('crypto_verify').Verify;
+
+function Sign(algorithm, options) {
```

Comment:
The `class` has been implemented in master, do u mind to change?

Reply:
But issue #483 is opened yet:(
if it was implemented, this pr is just my practice.

Reply:
What I mean is to use `class` constructor instead of prototype-based.

Reply:
akward! maybe i was misunderstand

Reply:
okay, this is just my practice😆